src/main/java/io/xstefank/wildfly/bot/util/GithubProcessor.java

```diff
@@ -232,19 +229,42 @@ public void sendEmail(String subject, String body, List<String> emails) {
         }
     }
 
-    public String skipPullRequest(GHPullRequest pullRequest, WildFlyConfigFile wildFlyConfigFile) throws IOException {
-        if (pullRequest.getBody() != null && SKIP_FORMAT_COMMAND.matcher(pullRequest.getBody()).find()) {
+    public String skipPullRequest(GHPullRequest pullRequest) throws IOException {
+        String body = pullRequest.getBody();
+        if (body != null && SKIP_FORMAT_COMMAND.matcher(body).find()) {
             return "skip format command found";
         }
 
+        if (pullRequest.isDraft()) {
+            return "pull request being a draft";
+        }
+
+        return null;
+    }
+
+    public String skipPullRequest(GHPullRequest pullRequest, WildFlyConfigFile wildFlyConfigFile) throws IOException {
```

Comment:
This was done so it can be called on both events. Which do and don't also retrieve `wildfly-bot.yml` file. Those that do would previously just return from such event if the received null as the config file.